docs/docs/setup/helm-deployment.md

```diff
@@ -0,0 +1,241 @@
+# Helm Chart Deployment
+
+Kuvasz provides an official Helm chart for deploying to Kubernetes clusters. This guide will walk you through the deployment process using Helm.
```

Comment:
same like above, when it's about the full name, use "Kuvasz Uptime" or "kuvasz-uptime", depending on the context